packages/common/src/store/ui/inbox-unavailable-modal/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Action } from '@reduxjs/toolkit'
+
+import { User } from 'models/User'
+
+import { createModal } from '../modals/createModal'
+
+export type InboxUnavailableModalState = {
+  user?: User
```

Comment:
does the whole user object need to be passed (guess I'll see further down in the PR but just commenting as I go hehe)?

Reply:
ooh very good point, we should probably just pass ID and fetch the user from the cache when needed